### PR TITLE
Remove outdated Homebrew instructions

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,16 +177,7 @@ TBD
 
 ### Installing and running on macOS via Homebrew
```
Comment (odeac) [inline README.md]: probably you have to remove this as well?

Or add a TBD after it, just like for the other incomplete sections

Comment (adaszko) [reply README.md]: Good catch.  Corrected.

**Comment by KellyatPyrofex** [comment]
@rabbitonweb, please confirm this change related to Mac users and Homebrew is correct.

File: README.md (hunk 1 above)
Comment (rabbitonweb) [reply README.md]: Adam just remove the whole Homebrew section from docs altogether

**Comment by adaszko** [comment]
@TomVasile Added a [Homebrew tap](https://github.com/rchain/homebrew-rchain/)

**Pull request closed without merging**